src/controllers/authenticationController.ts

```diff
@@ -20,6 +21,7 @@ export interface IPublicSession {
   readonly expired: boolean;
   readonly expireAt?: number;
   readonly newToken?: string;
+  readonly spidLevel?: SpidLevel;
```

Comment:
I think we should always send a value to the app, since the proxy manages the default value